### ISSUE TITLE
`no-standalone-expect` false negative

### ISSUE THREAD

**Issue by MathiasWP** [playwright-community/eslint-plugin-playwright#355]
If you import the "test" method and rename it to something else then this lint rule will throw an error.

Reproduction:

```ts
import { expect, test as setup } from "@playwright/test";

setup("set local state", async ({ page }) => {
  await expect(page.getByText("We use cookies")).toBeVisible(); // "Expect must be inside of a test block" is thrown 
  await page.getByTestId("cookie-settings").click();
})
```

**Comment by carlbray** [comment]
I think you need to add setup as an alias

https://github.com/playwright-community/eslint-plugin-playwright?tab=readme-ov-file#aliased-playwright-globals

**Comment by jeremy-daley-kr** [comment]
I was _just_ about to post about this same issue, but for a slightly different use case. We might potentially be adding a wrapper function which runs `test`, so that certain tests receive `tag` and `annotations` values, which are defined at a single source of truth, rather than needing to define them for every test across the codebase.

It would look something like:

```typescript
testAccessibility(`some test title`, ({ page }) => { ... });
```

The 2nd parameter is typed exactly how a regular `test` is (i.e. `TestBody<PlaywrightTestArgs & PlaywrightTestOptions`). I'm curious if the `@typescript-eslint/parser` is smart enough to know what the type of the parent function is, rather than what I assume is a simple string check for `'test'`.

**Comment by jeremy-daley-kr** [comment]
Food for thought... the Jest equivalent of this rule allows a rule config option:
[additionalTestBlockFunctions](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-standalone-expect.md#additionaltestblockfunctions)

**Comment by mskelton** [comment]
Closing as not planned.  We already support this: https://github.com/playwright-community/eslint-plugin-playwright#aliased-playwright-globals
